template/_dynamic_files.jinja2

```diff
@@ -910,6 +910,39 @@ jobs:
           file: "../../edit-comment.md"
           identifier: "UNREASONED"
 {% endif -%}
+{% if (('github_actions' in project.ci) and ('release-diff' in project.workflows)) %}
+^^^ .github/workflows/release-diff.yml
+name: Post release diff
+
+on:
+  # Triggers the workflow on pull request events for the {{ project.git_main_branch }} branch
+  pull_request:
+    branches: [ {{ project.git_main_branch }} ]
+    paths:
+      - 'src/ontology/reports/release-diff.md'
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+
+  post_diff:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Prepare release comment
+        env:
+            GITHUB_SHA: {% raw %}${{ github.sha }}{% endraw %}
+        run: "echo \"[Here's a diff of how this release impacts {{ project.id }}.owl](https://github.com/obophenotype/{{ project.repo }}/blob/${% raw %}${{ env.GITHUB_SHA }}{% endraw %}/src/ontology/reports/release-diff.md)\" >comment.md"
+      - name: Post reasoned comment
+        env:
+          GITHUB_TOKEN: {% raw %}${{ secrets.GITHUB_TOKEN }}{% endraw %}
+        uses: NejcZdovc/comment-pr@v2.0.0
```

Comment:
I may have made that comment in the past, if we have already agreed to take the risk, sorry :P we can move on. Just pointing out because of due dilligence.

Reply:
Found another one https://github.com/thollander/actions-comment-pull-request

Reply:
> I may have made that comment in the past

You have. This is the same action as the one already used in CL’s `gogoeditdiff` thing and Uberon’s taxon constraints check. I checked it; Shawn checked it; and I seem to recall that either Ugur or Huseyin checked it.

Reply:
yeah what @gouttegd said and we all checked it and couldnt find any obvious issue - I also couldnt find any better way to handle it last time.

Reply:
Yes, this action is suitable for posting more than one comment. I don't see a way to do it using the other one I just found.